packages/atomic/cypress/support/commands.ts

```diff
@@ -27,6 +28,9 @@ declare global {
       distanceTo(
         getSubjectB: () => Chainable<JQuery<HTMLElement>>
       ): Chainable<{horizontal: number; vertical: number}>;
+      getCalls<TArgs extends any[], TReturnValue>(): Chainable<
```

Comment:
Should shut down those warnings ;)

Reply:
I think it's important to still bring a lot of scrutiny on the usage of `any`, 'cause otherwise, well you end up with JavaScript.

There could be case-by-case ignore tho. (although preferably another type such as unknown should be used when possible)